mappings/net/minecraft/block/AbstractBlock.mapping

```diff
@@ -343,6 +348,7 @@ CLASS net/minecraft/unmapped/C_triydqro net/minecraft/block/AbstractBlock
 			ARG 3 block
 			ARG 4 posFrom
 			ARG 5 notify
+		METHOD m_hndkckmc spawnsBlockDustParticles ()Z
```

Comment:
```suggestion
		METHOD m_hndkckmc doesSpawnBlockDustParticles ()Z
```
[CONVENTIONS.md "getters"](https://github.com/QuiltMC/quilt-mappings/blob/1.20-pre6/CONVENTIONS.md#getters-setters-withers-and-creators)
> For boolean getters, use "is".

"is" is grammatically incorrect here -> "does"